CalibTracker/SiStripHitResolution/src/HitResol.cc

```diff
@@ -0,0 +1,651 @@
+////////////////////////////////////////////////////////////////////////////////
+// Package:          CalibTracker/SiStripHitResolution
+// Class:            HitResol
+// Original Authors: Denis Gele and Kathryn Coldham (adapted from HitEff)
+//                   modified by Khawla Jaffel for CPE studies
+//                   ported to cmssw by M. Musich
+//
+///////////////////////////////////////////////////////////////////////////////
+
+// system include files
+#include <memory>
+#include <string>
+#include <iostream>
+
+#include "CalibFormats/SiStripObjects/interface/SiStripDetCabling.h"
+#include "CalibFormats/SiStripObjects/interface/SiStripQuality.h"
+#include "CalibTracker/Records/interface/SiStripDetCablingRcd.h"
+#include "CalibTracker/Records/interface/SiStripQualityRcd.h"
+#include "CalibTracker/SiStripHitEfficiency/interface/SiStripHitEfficiencyHelpers.h"
+#include "CalibTracker/SiStripHitResolution/interface/HitResol.h"
+#include "CommonTools/Statistics/interface/ChiSquaredProbability.h"
+#include "DataFormats/Common/interface/DetSetVector.h"
```

Comment:
Not needed (also the "new")?

Reply:
done in https://github.com/cms-sw/cmssw/pull/40754/commits/01bbe6e597150b216293e2dc33cdada65a192d3d.